test/linsen.json

```diff
@@ -0,0 +1 @@
{"children":[],"direction":null,"format":"","indent":0,"type":"paragraph","version":1},{"children":[{"detail":0,"format":3,"mode":"normal","style":"","text":"Caesarsallad med kyckling och bacon/räkor","type":"text","version":1},{"detail":0,"format":0,"mode":"normal","style":"","text":" ","type":"text","version":1},{"detail":0,"format":2,"mode":"normal","style":"","text":"Caesar salad with chicken and bacon/shrimp","type":"text","version":1},{"detail":0,"format":0,"mode":"normal","style":"","text":" ","type":"text","version":1},{"detail":0,"format":3,"mode":"normal","style":"","text":"115:-/105:-","type":"text","version":1}],"direction":"ltr","format":"","indent":0,"type":"paragraph","version":1},{"children":[{"detail":0,"format":3,"mode":"normal","style":"","text":"Wokade nudlar med kyckling/tofu","type":"text","version":1},{"detail":0,"format":2,"mode":"normal","style":"","text":" Stir-fried noodles with chicken/tofu ","type":"text","version":1},{"detail":0,"format":3,"mode":"normal","style":"","text":"95:-/90:-","type":"text","version":1}],"direction":"ltr","format":"","indent":0,"type":"paragraph","version":1},{"children":[{"detail":0,"format":3,"mode":"normal","style":"","text":"En tallrik med sallad","type":"text","version":1},{"detail":0,"format":2,"mode":"normal","style":"","text":" A plate of salad","type":"text","version":1},{"detail":0,"format":0,"mode":"normal","style":"","text":" ","type":"text","version":1},{"detail":0,"format":3,"mode":"normal","style":"","text":"70:-","type":"text","version":1}],"direction":"ltr","format":"","indent":0,"type":"paragraph","version":1}],"direction":"ltr"}},"slug":"monday-menu","createdAt":"2024-04-19T17:07:00.008Z","updatedAt":"2024-05-27T08:49:31.008Z","richText_html":"<p>Meny V.22</p><p>27-05-2024</p><p><strong>Kalv Wallenbergare.</strong></p><p>Gröna Ärtor/ Skirat Smör/ Lingon/ Potatis Pure.</p><p><strong>Calf Wallenberger.</strong></p><p>Green Peas/ Clarified Butter/ Lingonberry/ Potato Puree.</p><p><strong>Halstrad Dagens Fångst.</strong></p><p>Dillmajo/ Pakchoi/ Ärtskott/ Lökstekta Potatis.</p><p><strong>Grilled Catch of the Day.</strong></p><p>Dill Mayo/ Pakchoi/ Pea Shoots/ Onion Fried Potatoes.</p><p><strong>Baingan Masala.</strong></p><p>Aubergine/ Paprika/ Svamp/ Raita/ Naanbröd/ Ris.</p><p><strong>Baingan Masala. </strong></p><p>Eggplant/ Paprika/ Mushroom/ Raita/ Naan bread/ Rice.</p><p><em><strong>115:-/105:-</strong></em></p><p></p><p><em><strong>Caesarsallad med kyckling och bacon/räkor</strong></em> <em>Caesar salad with chicken and bacon/shrimp</em> <em><strong>115:-/105:-</strong></em></p><p><em><strong>Wokade nudlar med kyckling/tofu</strong></em><em> Stir-fried noodles with chicken/tofu </em><em><strong>95:-/90:-</strong></em></p><p><em><strong>En tallrik med sallad</strong></em><em> A plate of salad</em> <em><strong>70:-</strong></em></p>"}],"totalDocs":6,"limit":10,"totalPages":1,"page":1,"pagingCounter":1,"hasPrevPage":false,"hasNextPage":false,"prevPage":null,"nextPage":null}
```

Comment:
This is horrible! :O 

That you got something well structured out of it impresses me.